df_chat/tests/test_chat.py

```diff
@@ -247,6 +249,88 @@ async def test_chat_multiple_rooms(self):
         await communicator2.disconnect()
         await communicator3.disconnect()
 
+    def test_create_message(self):
+        User = get_user_model()
```

Comment:
Could you please add a doc string that explains the steps required to create a message?

Reply:
Done!